### ISSUE TITLE
Bower build is broken

### ISSUE THREAD

**Issue by 0x54321** [Alex-D/Trumbowyg#1448]
### Description

  - OS: Mac Sonoma 14.2

When installing version 2.28.0 from Bower, the `dist` folder is missing.

### How to reproduce?

`bower install trumbowyg#2.28.0`

The installed `trumbowyg` directory does not include `dist`.

**Comment by Alex-D** [comment]
Yes, it would be best if you did not use Bower anymore. I will remove it from the documentation.
